MAUI/Expander/Appearance.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: Appearance in .NET MAUI Expander control | Syncfusion
+description: Learn here all about Appearance support in Syncfusion .NET MAUI Expander (SfExpander) control and more.
```

Comment:
Description: Learn about Appearance support in Syncfusion .NET MAUI Expander (SfExpander) control and more.

Reply:
Here, Description should be in small.